korge/src/commonMain/kotlin/com/soywiz/korge3d/RenderContext3D.kt

```diff
@@ -21,6 +21,5 @@ class RenderContext3D() {
 	val cameraMatInv: Matrix3D = Matrix3D()
 	val dynamicVertexBufferPool = Pool { ag.createVertexBuffer() }
     val dynamicIndexBufferPool = Pool { ag.createIndexBuffer() }
-	val ambientColor: Vector3D = Vector3D()
-
+	val ambientColor: Vector3D = Vector3D() // TODO: This shouldn't be a Vector3D
```

Comment:
It could be either:
https://github.com/korlibs/korim/blob/2dda8bacac23234a5429f860bbb447ae2f1ad134/korim/src/commonMain/kotlin/com/soywiz/korim/color/RGBAf.kt
Or:
https://github.com/korlibs/korim/blob/2dda8bacac23234a5429f860bbb447ae2f1ad134/korim/src/commonMain/kotlin/com/soywiz/korim/color/RGBA.kt

In 2D graphics, we are using multiplicative tinting with `colorMul: RGBA` and additive tinting with: `colorAdd: ColorAdd` both are packed as Int32, to the ColorAdd to support `[-255, +255]` range loses a bit of precission